phpmailer/phpmailer.php

```diff
@@ -0,0 +1,104 @@
+<?php
+/**
+ * Name: PHP Mailer SMTP
+ * Description: Connects to a SMTP server based on the config
+ * Version: 0.1
+ * Author: Marcus Mueller <http://mat.exon.name>
+ */
+
+use Friendica\App;
+use Friendica\Core\Addon;
+use PHPMailer\PHPMailer\PHPMailer;
+use PHPMailer\PHPMailer\Exception;
+
+function phpmailer_install()
+{
+	Addon::registerHook(
+		'emailer_send_prepare',
+		'addon/phpmailer/phpmailer.php',
```

Comment:
You can use the magic constant `__FILE__` instead of a hard-coded path here.

Reply:
Using `__FILE__` now.